Changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Next
 
+### Cleanup
```

Comment:
I'd prefer putting "Features/Enhancements" at the top. I'm also thinking if we want to have a separate Cleanup section. I'd like to keep it limited instead of adding too many sections. Items that have been cleaned up can go as "Deprecated xyz" in the "Features/Enhancement" section

Reply:
Makes sense, updating

Reply:
Thanks @mohammedzee1000.